### DISCUSSION TITLE
Do I need to be on the paid plan of OpenAI to access the GP model.

### DISCUSSION THREAD

**Discussion by vikramvee** [marcominerva/DatabaseGPT#63] [General]
When I tried to run the Web Project on my on premise DB, it gave the error>

`You exceeded your current quota, please check your plan and billing details.`

Does that mean that I need to be on the paid OpenAI plan?

**Comment by marcominerva** [comment]
Yes, you need a paid plain to use OpenAI APIs programmatically.
